### DISCUSSION TITLE
Syslog not working

### DISCUSSION THREAD

**Discussion by ByteAndBits** [Security-Onion-Solutions/securityonion#10895] [2.4]
OS 2.4 Beta4
Installed from ISO (yeah)
Standalone

Syslog not working. Made the Firewall entry, but nothing happend or show in Dashboard or Elatic.

Its the same syslog source and Hypervisor like for OS 2.3 where it worked perfect.
Syslog sender is in the same network.

**Comment by dougburks** [comment]
I've created an issue for this:
https://github.com/Security-Onion-Solutions/securityonion/issues/10896
